Content.Server/GameTicking/GameTicker.cs

```diff
@@ -53,8 +54,9 @@ public override void Initialize()
             InitializeLobbyMusic();
             InitializeLobbyBackground();
             InitializeGamePreset();
+            DebugTools.Assert(_prototypeManager.Index<JobPrototype>(FallbackOverflowJob).Name == Loc.GetString(FallbackOverflowJobName),
+                "Overflow role does not have the correct name!");
```

Comment:
this seems like a weird place for this to be

Reply:
Just me inlining InitializeJobController. Will move it.